packages/shared/src/components/cards/ShowMoreContent.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement, ReactNode, useState } from 'react';
+import { ClickableText } from '../buttons/ClickableText';
+
+interface ShowMoreContentProps {
+  content: string;
+  charactersLimit?: number;
+  threshold?: number;
+  contentPrefix?: ReactNode;
+  className?: string;
+}
+
+export default function ShowMoreContent({
+  content,
+  charactersLimit = 150,
+  threshold = 50,
+  contentPrefix,
+  className,
+}: ShowMoreContentProps): ReactElement {
+  const [isTextExpanded, setIsTextExpanded] = useState(false);
+  const linkName = isTextExpanded ? 'Show less' : 'Show more';
+
+  const toggleTextExpanded = () => {
+    setIsTextExpanded(!isTextExpanded);
+  };
+
+  const displayShowMoreLink = () => {
+    if (content && content?.length > charactersLimit + threshold) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
It looks cleaner in my opinion

```suggestion
  const displayShowMoreLink = () => content && content?.length > charactersLimit + threshold;
```